server/src/db/catalog/partition.rs

```diff
@@ -374,31 +386,44 @@ impl Partition {
 
     /// Return true if there are no other persisted chunks that are in the middle of
     /// the provided chunk orders
-    // NGA todo: There is test_compact_os_non_contiguous_chunks in
-    // compact_object_store.rs to test this but I will add more unit tests right here
-    // when PR #3167 ChunkGenerator is merged
-    pub fn contiguous_object_store_chunks(&self, chunk_orders: &BTreeSet<ChunkOrder>) -> bool {
-        // Last order in the chunk_orders for comparison
+    pub fn contiguous_chunks(
+        &self,
+        chunk_ids: &BTreeSet<ChunkId>,
+        chunk_orders: &BTreeSet<ChunkOrder>,
```

Comment:
```suggestion
        orders: RangeInclusive<ChunkOrder>,
```

The code path calling this is already computing the min order, it seems unnecessary to build up a BTreeSet only to generate a min and max